RocketChat/Rocket.Chat.ReactNative
# fix: enable android 13 notifications

## Proposed changes


Enable push notifications on android devices that are using android 13 or above.

TODO: Add an option somewhere so if the user refuses, he has the option to enable it again.

## Issue(s)	

resolve: #2385

## How to test or reproduce


- Test push notifications on Android 13/14 and (Android 12 or below)
- A lib was updated, so we need to test the AttachmentView as well (both iOS and Android)
   * Select a photo from your camera roll
   * send the photo
   * it should be sent correctly

## Screenshots

left android 14 - right android 12
![Screenshot 2023-06-22 at 18 27 50](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/18687097/f697f2b5-7730-4fc0-9227-31663264988b)

## Types of changes




## Checklist



## Further comments


Jira: [NATIVE-352]

[NATIVE-352]: https://rocketchat.atlassian.net/browse/NATIVE-352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
